### PR TITLE
[Kotlin Migration] AddNoteWidget 

### PR DESCRIPTION
**Pull request by codingtosh**
[Kotlin Migration] AddNoteWidget

### CODE REVIEW

**Review by david-allison-1 — APPROVED** [review]

File: AnkiDroid/src/main/java/com/ichi2/widget/AddNoteWidget.kt [hunk 1]
```diff
@@ -12,52 +12,38 @@
...
+        UsageAnalytics.sendAnalyticsEvent(this.javaClass.simpleName, "enabled")
     }
 
-
-    @Override
-    public void onDisabled(Context context) {
-        super.onEnabled(context);
-        UsageAnalytics.sendAnalyticsEvent(this.getClass().getSimpleName(), "disabled");
+    override fun onDisabled(context: Context) {
+        super.onEnabled(context)
```
Comment (Arthur-Milchior) [inline AnkiDroid/src/main/java/com/ichi2/widget/AddNoteWidget.kt:35]: Would you mind adding a note to check later. Calling onEnabled from onDisabled seems fishy. At least a comment would be required to explain what's going on.

Indeed, it's not a kotlin translation issue

**Review by Arthur-Milchior — APPROVED** [review]
A small request. Good to merge after that

File: AnkiDroid/src/main/java/com/ichi2/widget/AddNoteWidget.kt (hunk 1 above)
Comment (codingtosh) [reply AnkiDroid/src/main/java/com/ichi2/widget/AddNoteWidget.kt:40]: Done, PTAL.

**Review by mikehardy — APPROVED** [review]
nice

**Comment by mikehardy** [comment]
For this batch I'm putting them in the pending merge queue after approval, then pulling them locally to run a trial pull/rebase from master followed by all the CI checks. If it passes then I merge and move to the next. So the build errors here should be resolved by the previous build fix PR and I'll validate that before merging...

**Comment by mikehardy** [comment]
clean pass locally

**Comment by codingtosh** [comment]
> For this batch I'm putting them in the pending merge queue after approval, then pulling them locally to run a trial pull/rebase from master followed by all the CI checks. If it passes then I merge and move to the next. So the build errors here should be resolved by the previous build fix PR and I'll validate that before merging...

@mikehardy Thanks for the thorough validation :)

**Pull request merged**